Update staff-recruiting/README.md

natalie-hub opened the pull request:
Updated the Recruiting Process

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

natalie-hub committed:
Update staff-recruiting/README.md

Updated the Recruiting Process

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+In order to continually increase what we offer to customers we need to continually source, assess, hire and onboard new staff members with [high standards](https://handbook.mattermost.com/contributors/onboarding/staff-onboarding-guide#leadership-principles). The below outlines the process for recruiting new staff into the company.  
+
+## How to Open a New Role 
+Hiring Managers should confirm with your Team's MLT that your new headcount requisition has been approved. If not, your MLT team member should work with the Director of HR and VP Finance and Operations to gain approval. 
```
```suggestion
Hiring Managers should confirm with your team's MLT that your new headcount requisition has been approved. If not, your MLT team member should work with the Director of HR and VP of Finance and Operations to obtain approval. 
```

jasonblais replied:
Can people find out who is in the MLT from Bamboo, or should we link to https://handbook.mattermost.com/operations/operations/mlt-cadence#introduction-to-mlt?

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
-In order to continually increase what we offer to customers we need to continually source, assess, hire and onboard new staff members. The below outlines the process for recruiting new staff into the company. 
+In order to continually increase what we offer to customers we need to continually source, assess, hire and onboard new staff members with [high standards](https://handbook.mattermost.com/contributors/onboarding/staff-onboarding-guide#leadership-principles). The below outlines the process for recruiting new staff into the company.  
+
+## How to Open a New Role 
```
```suggestion
## How to open a new role 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+## How to Open a New Role 
+Hiring Managers should confirm with your Team's MLT that your new headcount requisition has been approved. If not, your MLT team member should work with the Director of HR and VP Finance and Operations to gain approval. 
 
 ## Role Description 
```
```suggestion
## Role description 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
 ## Role Description 
 
-Role descriptions should have clear responsibilities and requirements for each role that a recruiter can use to filter candidates and interviews can use to assess the candidate's fit for the role. 
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
```
```suggestion
Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the Recruiting Intake template.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
-Role descriptions should have clear responsibilities and requirements for each role that a recruiter can use to filter candidates and interviews can use to assess the candidate's fit for the role. 
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
+
+Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
```
```suggestion
Your Recruiter is assigned based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
+
+Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
+* Technical Individual Contributor Roles -- Teresa Novoa 
```
```suggestion
* Technical individual contributor roles - Teresa Novoa
```

justinegeffen replied:
```suggestion
* Technical individual contributor roles - Teresa Novoa
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
+* Technical Individual Contributor Roles -- Teresa Novoa 
+* All others -- Natalie Jew 
```
```suggestion
* All others - Natalie Jew
```

justinegeffen replied:
```suggestion
* All others - Natalie Jew
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+* Technical Individual Contributor Roles -- Teresa Novoa 
+* All others -- Natalie Jew 
+
+## Recruiter In-Take Session 
```
```suggestion
## Recruiter intake session
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+## Recruiter In-Take Session 
+
+Your Recruiter will schedule an in-take session after you’ve completed your Job Description. During this call, your Recruiter will work with you to ensure alignment on the level, ideal candidate profile, the interview process and next steps. 
```
```suggestion
Your Recruiter will schedule an Intake Session after you’ve completed your Job Description. During this call, your Recruiter will work with you to ensure alignment on the level, ideal candidate profile, the interview process, and next steps.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+Your Recruiter will schedule an in-take session after you’ve completed your Job Description. During this call, your Recruiter will work with you to ensure alignment on the level, ideal candidate profile, the interview process and next steps. 
+
+### Role Leveling 
```
```suggestion
### Role leveling
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+### Role Leveling 
+
+You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). Note: HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job req, your recruiter can help you walk through this as part of the Recruiter In-Take. 
```
```suggestion
You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). 

**Note:** HR is also working with MLT to build v2 which will help provide additional clarity. If you're unsure about how to use or assign a level to a new job requisition, your Recruiter can help you walk through this as part of the Recruiter Intake.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). Note: HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job req, your recruiter can help you walk through this as part of the Recruiter In-Take. 
+
+### Ideal Candidate Profile (ICP) 
```
```suggestion
### Ideal Candidate Profile (ICP) 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+### Ideal Candidate Profile (ICP) 
+
+The ICP should be a clear list of key skills, experiences and/or mindsets that the Hiring Manager believes represent the Ideal Candidate. The Hiring Manager will be asked to stack rank the ICP list and use ‘P0’ to indicate "must haves" and 'P1' to indicate "nice to have" the ideal skills, experiences and mindsets. 
```
```suggestion
The ICP should be a clear list of key skills, experiences, and/or mindsets that the Hiring Manager believes represent the Ideal Candidate. The Hiring Manager will be asked to stack-rank the ICP list and use "P0" to indicate "must-haves" and "P1" to indicate "nice-to-have" the ideal skills, experiences, and mindsets.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+The ICP should be a clear list of key skills, experiences and/or mindsets that the Hiring Manager believes represent the Ideal Candidate. The Hiring Manager will be asked to stack rank the ICP list and use ‘P0’ to indicate "must haves" and 'P1' to indicate "nice to have" the ideal skills, experiences and mindsets. 
+
+The ICP will be used to align the sourcing, interview panel focus areas and onboarding goals for this new role. For example, here’s an ICP for a [Lead Customer Journey Engineer](https://jobs.lever.co/mattermost/3c1dd4ff-b135-4bb7-8b14-73c4a8cd674f). The list below reflects the top skills and experience to source for candidates, the area of focus for the interview panel and what the eventual new h ...
```
```suggestion
The ICP will be used to align the sourcing, interview panel focus areas, and onboarding goals for this new role. For example, here’s an ICP for a [Lead Customer Journey Engineer](https://jobs.lever.co/mattermost/3c1dd4ff-b135-4bb7-8b14-73c4a8cd674f). The list below reflects the top skills and experience to source for candidates, the area of focus for the interview panel, and what the eventual new hire's onboarding goals will be based on.  
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+The ICP will be used to align the sourcing, interview panel focus areas and onboarding goals for this new role. For example, here’s an ICP for a [Lead Customer Journey Engineer](https://jobs.lever.co/mattermost/3c1dd4ff-b135-4bb7-8b14-73c4a8cd674f). The list below reflects the top skills and experience to source for candidates, the area of focus for the interview panel and what the eventual new h ...
 
-**Responsibilities:** 
+P0 - 2+ Years experience leading a team of engineers
```
```suggestion
**P0:** 2+ years of experience leading a team of engineers
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
 
-**Responsibilities:** 
+P0 - 2+ Years experience leading a team of engineers
+P1 - Experience as an Engineer or a Lead for a team doing front-end/UI-UX work in JavaScript, React, etc.
```
```suggestion
**P1:** Experience as an Engineer or a Lead for a team doing front-end/UI/UX work in JavaScript, React, etc.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+### Changing the ICP 
 
-**Requirements:** 
+The ICP is also a helpful indicator of talent pools. If a role is not gaining enough candidate pipeline or the candidate fit is off, the ICP enables Hiring Managers to efficiently determine appropriate tradeoffs as it relates to the P0 or P1 skills, experience and mindsets. For example, a Recruiter may find that we have filtered multiple candidates who do not have a specific P0 skill. As a result ...
```
```suggestion
The ICP is also a helpful indicator of talent pools. If a role is not gaining enough candidate pipeline or the candidate fit is off, the ICP enables Hiring Managers to efficiently determine appropriate tradeoffs as it relates to the "P0" or "P1" skills, experience, and mindsets. 

For example, a Recruiter may find that we have filtered multiple candidates who do not have a specific "P0" skill. As a result, a Hiring Manager may determine that it may be worth deprioritizing a specific qualification from "P0" to "P1" in order to widen the candidate pool. If an ICP is ever changed, the Recruiter will ensure that the hiring team is aligned to the change.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
-* What you want recruiters to screen for in providing candidates? 
-* What will you want your interview panel to verify in terms of core capabilities \(e.g. ability to hire and manage a remote-first team, etc.\)? 
+## Pre-Interview Screens
+Hiring Managers will review the Pre-Interview Screening process during the Recruiter In-Take Session. 
```
```suggestion
Hiring Managers will review the Pre-Interview Screening process during the Recruiter Intake Session. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+## Pre-Interview Screens
+Hiring Managers will review the Pre-Interview Screening process during the Recruiter In-Take Session. 
+
+### Resume Screen 
```
```suggestion
### Resume screen
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+### Resume Screen 
+
+Hiring Managers will review resumes and tag their Recruiter in Lever to indicate interest in a candidate. Hiring Managers can review Candidates that are in the Shortlist or New Lead stage. 
```
```suggestion
Hiring Managers will review resumes and tag their Recruiter in Lever to indicate interest in a candidate. Hiring Managers can review candidates that are in the **Shortlist** or **New Lead** stage. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+Hiring Managers will review resumes and tag their Recruiter in Lever to indicate interest in a candidate. Hiring Managers can review Candidates that are in the Shortlist or New Lead stage. 
+
+The Recruiter will also archive Candidates that are located in regions we are not able to hire or do not match the location target of the role. 
```
```suggestion
The Recruiter will also archive candidates that are located in countries/regions we are not able to hire or do not match the location target of the role. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+The Recruiter will also archive Candidates that are located in regions we are not able to hire or do not match the location target of the role. 
+
+### Post-Application Candidate Questions \(Optional\) 
```
```suggestion
### Post-Application candidate questions \(optional\) 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+### Post-Application Candidate Questions \(Optional\) 
+
+This is an optional step, but is highly recommended to improve candidate & job match ahead of any reach out. Hiring Managers can create a set of questions that every candidate will be asked to complete either as part of the application process or before the Recruiter Screen. 
```
```suggestion
This is an optional step, but is highly recommended to improve candidate and job match ahead of any reach out. Hiring Managers can create a set of questions that every candidate will be asked to complete either as part of the application process or before the Recruiter Screen.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+This is an optional step, but is highly recommended to improve candidate & job match ahead of any reach out. Hiring Managers can create a set of questions that every candidate will be asked to complete either as part of the application process or before the Recruiter Screen. 
+
+### Recruiter Screen \(Optional\) 
```
```suggestion
### Recruiter Screen \(optional\) 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+* Have one main point of contact throughout the interview process 
+* Collect compensation requirements and verify location of residence 
+* Provide the candidate with more information on: Mattermost, the role, the recruiting process, and to understand the candidate’s: timeline, role/career aspirations, assess interest, compensation expectations and answer questions.
+* Hiring Managers can also request the Recruiter to ask specific questions as part of the Recruiter Screen. Hiring Managers can decide the set of questions during the Recruiter in-take.
```
```suggestion
* Hiring Managers can also request the Recruiter to ask specific questions as part of the Recruiter Screen. Hiring Managers can decide the set of questions during the Recruiter Intake.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+### Recruiter Screen \(Optional\) 
+
+Hiring Managers can customize the questions asked during the Recruiter screen or they may decide to remove this stage and handle the first screen themselves. The purpose of this screen is to:
+* Have one main point of contact throughout the interview process 
```
```suggestion
* Have one main point of contact throughout the interview process.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+Hiring Managers can customize the questions asked during the Recruiter screen or they may decide to remove this stage and handle the first screen themselves. The purpose of this screen is to:
+* Have one main point of contact throughout the interview process 
+* Collect compensation requirements and verify location of residence 
```
```suggestion
* Collect compensation requirements and verify country/region of residence.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+Hiring Managers can customize the questions asked during the Recruiter screen or they may decide to remove this stage and handle the first screen themselves. The purpose of this screen is to:
+* Have one main point of contact throughout the interview process 
+* Collect compensation requirements and verify location of residence 
+* Provide the candidate with more information on: Mattermost, the role, the recruiting process, and to understand the candidate’s: timeline, role/career aspirations, assess interest, compensation expectations and answer questions.
```
```suggestion
* Provide the candidate with more information on: Mattermost, the role, the recruiting process, and to understand the candidate’s: timeline, role/career aspirations, assess interest, compensation expectations, and answer questions.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+* Provide the candidate with more information on: Mattermost, the role, the recruiting process, and to understand the candidate’s: timeline, role/career aspirations, assess interest, compensation expectations and answer questions.
+* Hiring Managers can also request the Recruiter to ask specific questions as part of the Recruiter Screen. Hiring Managers can decide the set of questions during the Recruiter in-take.
 
 ## Pre-Meet Scores \(Optional\) 
```
```suggestion
## Pre-meet scores \(optional\) 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
 * **Gender identity other than male \(+3\)**
 * **Lack of any university of college education \(+3\):** For someone to be highly successful in technology without a formal education is rare and team members will this profile can help us expand into new talent pools
 
+## Interview Process 
```
```suggestion
## Interview process
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
 
+## Interview Process 
+
+Interview panels and the order of interviewers will be determined by the Hiring Manager during the Recruiter In-Take. Ahead of the first interview for a new req, the Recruiter will send the Interview Team a summary of the Interview Plan, including each Interviewer's Focus Area(s). Hiring Manager will also indicate the Interview Mid-Point which will also drive the Mid-Interview Syncs. 
```
```suggestion
Interview panels and the order of interviewers is determined by the Hiring Manager during the Recruiter Intake. Ahead of the first interview for a new requisition, the Recruiter will send the Interview Team a summary of the Interview Plan, including each interviewer's focus area(s). Hiring Manager will also indicate the Interview Mid-Point which will also drive the Mid-Interview Syncs.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+Interview panels and the order of interviewers will be determined by the Hiring Manager during the Recruiter In-Take. Ahead of the first interview for a new req, the Recruiter will send the Interview Team a summary of the Interview Plan, including each Interviewer's Focus Area(s). Hiring Manager will also indicate the Interview Mid-Point which will also drive the Mid-Interview Syncs. 
+
+For the purpose of a positive candidate experience and efficient use of mana, it's recommended that there be no more than 4 non-Hiring Manager and non-MLT interviewers. Ultimately, the Hiring Manager will decide how many interviewers are needed. 
```
```suggestion
For the purpose of a positive candidate experience and efficient use of mana, it's recommended that there be no more than four non-Hiring Manager and non-MLT interviewers. Ultimately, the Hiring Manager will decide how many interviewers are needed.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+For the purpose of a positive candidate experience and efficient use of mana, it's recommended that there be no more than 4 non-Hiring Manager and non-MLT interviewers. Ultimately, the Hiring Manager will decide how many interviewers are needed. 
+
+### Interview Panel Process
+As part of the In-take Session, the Recruiter will also work with the Hiring Manager to define the scope and focus area(s) for each interviewer using the ICP. Ahead of the first candidate interview, the Recruiter will post an update in the Recruiting Updates Channel and tag the Interview Panel. The update will include  the scope and focus area(s) for each interviewer. To promote [high standards]( ...
```
```suggestion
As part of the Recruiter Intake Session, the Recruiter will also work with the Hiring Manager to define the scope and focus area(s) for each interviewer using the ICP. Ahead of the first candidate interview, the Recruiter will post an update in the Recruiting Updates Channel and tag the Interview Panel. 

The update will include the scope and focus area(s) for each interviewer. To promote [high standards](https://handbook.mattermost.com/contributors/onboarding/staff-onboarding-guide#leadership-principles) hiring, if more than two interviewers are not a **Strong Yes** ahead of the Mid-Interview Sync, the candidate will not move forward. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+### Recruiter Screen \(Optional\) 
+
+Hiring Managers can customize the questions asked during the Recruiter screen or they may decide to remove this stage and handle the first screen themselves. The purpose of this screen is to:
```
```suggestion
Hiring Managers can customize the questions asked during the Recruiter Screen or they may decide to remove this stage and handle the first screen themselves. The purpose of this screen is to:
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+1. Interviewers should submit their 1-4 rating in Lever within 1 hour of the interview completion. 
+2. Interviews should submit their full summary feedback within 24 hours of the interview. 
+
+**Tips on Sharing Candidate Feedback** 
```
```suggestion
**Tips on sharing candidate feedback** 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+2. Interviews should submit their full summary feedback within 24 hours of the interview. 
+
+**Tips on Sharing Candidate Feedback** 
+* Required: Lever’s 1-4 rating on the candidate 
```
```suggestion
* **Required:** Lever’s 1-4 rating on the candidate.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+**Tips on Sharing Candidate Feedback** 
+* Required: Lever’s 1-4 rating on the candidate 
+* Clearly state your opinion on how this candidate ranks compared to other candidates interviewed as it relates to your focus area. This should be objective and simple. For example: Strong Yes for SRE, but not quite as technical in Kubernetes as John Doe. 
```
```suggestion
* Clearly state your opinion on how this candidate ranks compared to other candidates interviewed as it relates to your focus area. This should be objective and simple. For example: "Strong Yes for SRE, but not quite as technical in Kubernetes as John Doe."
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+**Tips on Sharing Candidate Feedback** 
+* Required: Lever’s 1-4 rating on the candidate 
+* Clearly state your opinion on how this candidate ranks compared to other candidates interviewed as it relates to your focus area. This should be objective and simple. For example: Strong Yes for SRE, but not quite as technical in Kubernetes as John Doe. 
+* Feedback on the specific skill/focus area assigned with examples to help contextualize your rating  
```
```suggestion
* Feedback on the specific skill/focus area assigned with examples to help contextualize your rating.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+* Required: Lever’s 1-4 rating on the candidate 
+* Clearly state your opinion on how this candidate ranks compared to other candidates interviewed as it relates to your focus area. This should be objective and simple. For example: Strong Yes for SRE, but not quite as technical in Kubernetes as John Doe. 
+* Feedback on the specific skill/focus area assigned with examples to help contextualize your rating  
+* Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
```
```suggestion
* Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown M&Ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+* Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
+
+### Mid-Interview Sync
+Ahead of MLT and AA interviews, the Recruiter may schedule a live mid-interview sync to align on intent to move forward and any ICP criteria that need additional assessment.  If there are multiple candidates in the pipe, the Recruiter may ask the interview panel to rank the candidate pool. This exercise helps clear any potential blindspots or confusion around role expectations. It also allows the ...
```
```suggestion
Ahead of MLT and AA interviews, the Recruiter may schedule a live Mid-Interview Sync to align on intent to move forward and any ICP criteria that need additional assessment. 

If there are multiple candidates in the pipe, the Recruiter may ask the interview panel to rank the candidate pool. This exercise helps clear any potential blindspots or confusion around role expectations. It also allows the Recruiting team to prioritize the highest ranking candidates to complete the interview process, as appropriate.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+Ahead of MLT and AA interviews, the Recruiter may schedule a live mid-interview sync to align on intent to move forward and any ICP criteria that need additional assessment.  If there are multiple candidates in the pipe, the Recruiter may ask the interview panel to rank the candidate pool. This exercise helps clear any potential blindspots or confusion around role expectations. It also allows the ...
+
+**Logistics**
+* Mid-interview sync will be pre-scheduled to occur after the last mid-interviewer completes. 
```
```suggestion
* The Mid-Interview Sync will be pre-scheduled to occur after the last mid-interviewer completes. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+**Logistics**
+* Mid-interview sync will be pre-scheduled to occur after the last mid-interviewer completes. 
+* If there are multiple candidates for the same role, the sync may be scheduled after all candidates have gone through the last mid-interviewer. 
```
```suggestion
* If there are multiple candidates for the same role, the sync may be scheduled after all candidates have gone through the last mid-interviewer.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+**Logistics**
+* Mid-interview sync will be pre-scheduled to occur after the last mid-interviewer completes. 
+* If there are multiple candidates for the same role, the sync may be scheduled after all candidates have gone through the last mid-interviewer. 
+* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync. 
```
```suggestion
* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync. 
+
+### Interview Project
+Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the in-take. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
```
```suggestion
Candidates may be asked to complete a take-home project as part of the interview process. Hiring Managers will indicate this need during the Recruiting Intake. Hiring Managers can create their own project or reach out to Recruiting for sample projects.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+### Interview Project
+Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the in-take. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
+
+All take home projects should list the expected amount of time to complete. This provides candidates with expectation clarity. The Hiring Manager will indicate which stage of the recruiting process a take home will be given. The process for this stage will be standard for all roles: 
```
```suggestion
All take-home projects should list the expected amount of time to complete. This provides candidates with expectation clarity. The Hiring Manager will indicate which stage of the recruiting process a take-home will be given. The process for this stage will be standard for all roles:
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+All take home projects should list the expected amount of time to complete. This provides candidates with expectation clarity. The Hiring Manager will indicate which stage of the recruiting process a take home will be given. The process for this stage will be standard for all roles: 
+
+1. Hiring Manager or Recruiter will inform the candidate of the Take Home project details and ask the candidate to complete either: 
```
```suggestion
1. Hiring Manager or Recruiter will inform the candidate of the take-home project details and ask the candidate to complete either:
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+All take home projects should list the expected amount of time to complete. This provides candidates with expectation clarity. The Hiring Manager will indicate which stage of the recruiting process a take home will be given. The process for this stage will be standard for all roles: 
+
+1. Hiring Manager or Recruiter will inform the candidate of the Take Home project details and ask the candidate to complete either: 
+ * [CLA](https://mattermost.org/mattermost-contributor-agreement/)if the Take Home project is a Community Ticket 
```
```suggestion
 * [CLA](https://mattermost.org/mattermost-contributor-agreement/) if the take-home project is a Community ticket.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+1. Hiring Manager or Recruiter will inform the candidate of the Take Home project details and ask the candidate to complete either: 
+ * [CLA](https://mattermost.org/mattermost-contributor-agreement/)if the Take Home project is a Community Ticket 
+ * NDA if the above doesn’t apply 
```
```suggestion
 * NDA if the above doesn’t apply.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+1. Hiring Manager or Recruiter will inform the candidate of the Take Home project details and ask the candidate to complete either: 
+ * [CLA](https://mattermost.org/mattermost-contributor-agreement/)if the Take Home project is a Community Ticket 
+ * NDA if the above doesn’t apply 
+2. Hiring Manager or Recruiter will send the candidate the Take Home instructions through Lever. 
```
```suggestion
2. Hiring Manager or Recruiter will send the candidate the take-home project instructions through Lever.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+ * [CLA](https://mattermost.org/mattermost-contributor-agreement/)if the Take Home project is a Community Ticket 
+ * NDA if the above doesn’t apply 
+2. Hiring Manager or Recruiter will send the candidate the Take Home instructions through Lever. 
+3. Upon completion, the Hiring Manager or Recruiter will complete the [Interview Project Gift Card Form](https://docs.google.com/forms/d/e/1FAIpQLSfMppjsYmEyrMmY3tXnKdAs_cvA21o0vHRcyuVN6uFOoU3_Eg/viewform) which will notify the People Team to send a Thank You gift to the candidate for completing the project. 
```
```suggestion
3. Upon completion, the Hiring Manager or Recruiter will complete the [Interview Project Gift Card Form](https://docs.google.com/forms/d/e/1FAIpQLSfMppjsYmEyrMmY3tXnKdAs_cvA21o0vHRcyuVN6uFOoU3_Eg/viewform) which will notify the People Team to send a Thank You gift to the candidate for completing the project.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+2. Hiring Manager or Recruiter will send the candidate the Take Home instructions through Lever. 
+3. Upon completion, the Hiring Manager or Recruiter will complete the [Interview Project Gift Card Form](https://docs.google.com/forms/d/e/1FAIpQLSfMppjsYmEyrMmY3tXnKdAs_cvA21o0vHRcyuVN6uFOoU3_Eg/viewform) which will notify the People Team to send a Thank You gift to the candidate for completing the project. 
+
+## Offer Stage
```
```suggestion
## Offer stage
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+## Offer Stage
+
+### Reference Checks 
```
```suggestion
### Reference checks 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+### Reference Checks 
+WIP 
+
+### Offer Approval Process 
```
```suggestion
### Offer approval process 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+WIP 
+
+### Offer Approval Process 
+Once reference checks are complete and an offer is ready to be made. The Recruiter will enter in the offer information in Lever for approval. 
```
```suggestion
Once reference checks are complete and an offer is ready to be made. The Recruiter will enter in the offer information in Lever for approval.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+### Offer Approval Process 
+Once reference checks are complete and an offer is ready to be made. The Recruiter will enter in the offer information in Lever for approval. 
+
+The order of approval will follow: 
```
```suggestion
The order of approval will follow:
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+Once reference checks are complete and an offer is ready to be made. The Recruiter will enter in the offer information in Lever for approval. 
+
+The order of approval will follow: 
+1. HR Director 
```
```suggestion
1. HR Director
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+The order of approval will follow: 
+1. HR Director 
+2. MLT team member 
```
```suggestion
2. MLT member
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+The order of approval will follow: 
+1. HR Director 
+2. MLT team member 
+3. VP, Finance & Operations 
```
```suggestion
3. VP of Finance and Operations
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+1. HR Director 
+2. MLT team member 
+3. VP, Finance & Operations 
+4. CEO 
```
```suggestion
4. CEO
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+3. VP, Finance & Operations 
+4. CEO 
+
+After offer approval, the Recruiter, Team MLT or AA Interview will extend a verbal offer to the candidate. Upon verbal acceptance, the Director of HR will send the offer letter. 
```
```suggestion
After offer approval, the Recruiter, Team MLT member, or AA Interviewer will extend a verbal offer to the candidate. Upon verbal acceptance, the Director of HR will send the offer letter.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+After offer approval, the Recruiter, Team MLT or AA Interview will extend a verbal offer to the candidate. Upon verbal acceptance, the Director of HR will send the offer letter. 
+
+### Extending an Offer 
```
```suggestion
### Extending an offer 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+### Extending an Offer 
+
+Recruiting will typically extend an offer, unless otherwise determined by the MLT team member or Hiring Manager. Exceptions may be made if the Team MLT memberor AA Interviewer prefers to extend the offer. 
```
```suggestion
Recruiting will typically extend an offer, unless otherwise determined by the MLT member or Hiring Manager. Exceptions may be made if the Team MLT member or AA Interviewer prefers to extend the offer.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+Recruiting will typically extend an offer, unless otherwise determined by the MLT team member or Hiring Manager. Exceptions may be made if the Team MLT memberor AA Interviewer prefers to extend the offer. 
+
+### Sending an Offer & Notifying Internal Teams 
```
```suggestion
### Sending an offer and notifying internal teams 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+### Sending an Offer & Notifying Internal Teams 
+This process will be managed by HR. 
+
+1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence region and send for signatures. 
```
```suggestion
1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence country/region and send for signatures.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+This process will be managed by HR. 
+
+1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence region and send for signatures. 
+2. Upon full execution of a new Staff contract, the Recruiter will mark the candidate as “hired” in Lever and export the profile to BambooHR. 
```
```suggestion
2. Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever and export the profile to BambooHR.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence region and send for signatures. 
+2. Upon full execution of a new Staff contract, the Recruiter will mark the candidate as “hired” in Lever and export the profile to BambooHR. 
+3. Once exported to BambooHR, an automatic New Starter Notification will be sent to the Hiring Team to kick-off the onboarding process. 
```
```suggestion
3. Once exported to BambooHR, an automatic New Starter Notification will be sent to the Hiring Team to kick-off the onboarding process.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+2. Upon full execution of a new Staff contract, the Recruiter will mark the candidate as “hired” in Lever and export the profile to BambooHR. 
+3. Once exported to BambooHR, an automatic New Starter Notification will be sent to the Hiring Team to kick-off the onboarding process. 
+
+### Pre-Onboarding Process
```
```suggestion
### Pre-onboarding process
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+3. Once exported to BambooHR, an automatic New Starter Notification will be sent to the Hiring Team to kick-off the onboarding process. 
+
+### Pre-Onboarding Process
+New Colleagues will receive pre-onboarding instructions from the People team. Hiring Managers will also receive a request to indicate new systems for their new direct report. 
```
```suggestion
New colleagues will receive pre-onboarding instructions from the People team. Hiring Managers will also receive a request to indicate new systems for their new direct report.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+### Pre-Onboarding Process
+New Colleagues will receive pre-onboarding instructions from the People team. Hiring Managers will also receive a request to indicate new systems for their new direct report. 
+* The Recruiter will share the ICP with HR as part of the Offer Request. 
```
```suggestion
* The Recruiter will share the ICP with HR as part of the Offer Request.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+### Pre-Onboarding Process
+New Colleagues will receive pre-onboarding instructions from the People team. Hiring Managers will also receive a request to indicate new systems for their new direct report. 
+* The Recruiter will share the ICP with HR as part of the Offer Request. 
+* Hiring Managers should review the role ICP with the new hire during Onboarding to help align new colleagues to the role’s focus areas and help contextualize a new colleague’s priorities as they ramp-up with the team and their new peers.
```
```suggestion
* Hiring Managers should review the role ICP with the new hire during Onboarding to help align them to the role’s focus areas and help contextualize their priorities as they ramp-up with the team and their new peers.
```

justinegeffen commented:
Thanks, @natalie-hub.

I may have missed one or two instances of terminology so once you've had a chance to go through this please LMK and I can give it another pass to confirm it's all aligned stylistically. :)

jasonblais reviewed (dismissed):
Overall looks good, just added one question, non-blocking. No other feedback than what Justine already shared :)

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update README.md

Added the link for MLT that Jason recommended.

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub reviewed (commented):
Thanks Justine, I reviewed and all the changes look good! I made one update to the link to MLT that JB shared. Thanks again!

natalie-hub committed:
Create Recruiting Cadences (#339)

* Create Recruiting Cadences

Created a Recruiting Cadence to help outline the Manager's responsibilities a bit more clearly.

* Update and rename Recruiting Cadences to recruiting-cadences.md

First, full editorial pass for new section.

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais commented:
@justinegeffen I think this one is ready to merge?
